lib/main.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+import 'package:mobile_test/screens/home_screen.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+    
+      ),
```

Comment:
**question:** why was an empty theme provided? Would it behave the same if one wasn't provided?

Reply:
It had deprecated properties, so I went ahead and removed those. But yes, it would behave the same if no properties were provided to as if it would. (If I had no color defined in AppBar in each screen, it would look into themeData provided and use that) 
On a large-scale application, it's would be efficient to have ThemeData defined so that by changing value at a single place it can take effect everywhere.